src/test/java/pages/MainPage.java

```diff
@@ -19,24 +20,56 @@
     @FindBy(xpath = "//form/descendant::div[@class='auth-form__label-title']")
     public Text loginViaNickNameMessage;
 
-    @FindBy(className = "cart-form__title_condensed-additional")
-    public Text shoppingCartMessage;
+    @FindBy(linkText = "Зарегистрироваться на Onlíner")
+    public Link registrationLink;
+
+    @FindBy(xpath = "//input[@placeholder='Ник или e-mail']")
+    public Input loginInput;
+
+    @FindBy(xpath = "//input[@type='password']")
+    public Input passwordInput;
+
+    @FindBy(xpath = "//div/button[@type='submit']")
+    public Button loginButton;
+
+    @FindBy(className = "js-header-user-avatar")
+    public Picture avatar;
+
+    @FindBy(xpath = "//ul/li[@class='b-main-navigation__item']/a[2]/span")
+    public Link catalogue;
+
+    @FindBy(xpath = "//footer//ul/li[1]/a")
+    public Link aboutCompany;
+
+    @FindBy(css = ".auth-form__description.auth-form__description_extended-other")
+    public Text incorrectCredentialsWarning;
+
+    @FindBy(xpath = "//*[contains(text(),'Укажите пароль')]")
+    public Text emptyPasswordWarning;
+
+    @FindBy(xpath = "//form/input[@name='query']")
+    public Input mainSearchField;
+
+    @FindBy(css = ".search__suggest-addon.search__suggest-addon_active")
+    public Text searchInputHint;
 
     public MainPage() {
         super();
     }
 
     @Override
     public MainPage openPage() {
-        driver.navigate().to(properties.getPropertyValueByKey("url"));
+        driver.navigate().to(props.getKeyProperty("url"));
+        logger.debug("Navigation to the url...");
```

Comment:
I think it makes sense to specify url details in this logger. For all the similar cases
When we read logs we want to understand what was happening in the program but when we see ''navigating to the url" we don't understand to which url

Reply:
done